.github/workflows/helm_build_release.yaml

```
@@ -3,16 +3,31 @@ name: Build and release helm chart
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - '.devcontainer/**'
+      - '.github/**'
+      - '.test/**'
+      - 'README.md'
+      - 'LICENSE'
 
 jobs:
   helm-release:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
+      - name: Get chart version
+        id: get_chart_version
+        uses: Energinet-Datahub/.github/.github/actions/yaml-get@main
+        with:
+          yaml_file: chart/Chart.yaml
+          yaml_path: version
+
       - name: Release helm chart
-        uses: Energinet-DataHub/.github/.github/actions/helm-release-chart@helm_releases_action
+        uses: Energinet-DataHub/.github/.github/actions/helm-release-chart@5.0.0
         with:
-          token: ${{ secrets.ACCESS_TOKEN }}
+          dispatch_token: ${{ secrets.ACCESS_TOKEN }}
+          release_token: ${{ secrets.ACCESS_TOKEN }}
           chart_folder: chart
           chart_repository: Energinet-DataHub/helm-charts
+          chart_version: ${{ steps.get_chart_version.outputs.result }}
```

Pull main into the branch, these lines where already changed.
Further chart_version is only nessisary when overriding what is already stated in the Chart.yaml file, the line is redundant.

```suggestion
          release_token: ${{ secrets.RELEASE_CHART_TOKEN }}
          dispatch_token: ${{ secrets.RELEASE_CHART_TOKEN }}
          chart_folder: chart
          chart_repository: Energinet-DataHub/helm-charts
```